chia/util/keychain.py

```diff
@@ -264,7 +264,7 @@ def _get_pk_and_entropy(self, user: str) -> Optional[Tuple[G1Element, bytes]]:
             return None
         str_bytes = bytes.fromhex(read_str)
         return (
-            G1Element.from_bytes(str_bytes[: G1Element.SIZE]),
+            G1Element.from_bytes_unchecked(str_bytes[: G1Element.SIZE]),
```

Comment:
do we call this function often enough for this optimization to matter?

Reply:
the wallet might call this many times, but it probably does not matter